index.html

```diff
@@ -203,6 +198,39 @@ <h2>Debug Informations</h2>
         <h1> Move Timer: </h1>
         <progress value="0" max="2.5" id="timer-bar"></progress>
     </div>
+    <script>{
```

Comment:
Let's not put this script inline, since no other scripts are inline. Instead, please move this to dom_events.js